### ISSUE TITLE
Working with OTU table (Relative abundance)

### ISSUE THREAD

**Issue by Jayanth-kumar5566** [zdk123/SpiecEasi#127]
Hi,
Is it possible to implement spiec.easi using an OTU table (that is in a relative abundance format)?

The documentation reads "For a matrix, non-normalized count OTU/data table with samples on rows and features/OTUs in columns"

**Comment by zdk123** [comment]
Is your use case for non-count data? We typically want to make sure users use pseudo-counts directly.

**Comment by Jayanth-kumar5566** [comment]
Yes, right. My use case is for non-count data. 
So, what you mean by pseudo-counts; I multiply my relative abundance values by some constant number (say 100) and round these values to the nearest integer?

**Comment by zdk123** [comment]
no: because of the preponderance of zeros in microbiome count data, we have to add a one (the pseudo-count) to [estimate the relative abundances](https://en.wikipedia.org/wiki/Additive_smoothing).

It seems like you have relative abundances without any zeros, and want to skip the pseudocount?

**Comment by guangingmai** [comment]
Hi zdk123,
My count otu table is rarefying the same depth for every sample, and the rarefied otu table is also with count in it. So should i use my rarefied count otu table for network analysis or use otu table but with relative abundance for network analysis? And should i generate a synthetic network before network inference analysis?

**Comment by zdk123** [comment]
@guangingmai we don't recommend using rarefied count data... go back to raw couts.

**Comment by mahmhmmdi** [comment]
> 

Hi @zdk123 
I'm not sure if I understand correctly, but do we have to add 1 to all relative abundance data in order to make pseudo-count data and then use that as input?
I have another question which is if there is a minimum for time points that we should use. Or the minimum is for the number of OTUs,  not time points? 
My data consist of 7 days of sampling and each day has about 2400 OTUs. Can I run SPIEC-EASI for each day or two?
